firstvoices/backend/serializers/media_serializers.py

```diff
@@ -74,13 +74,22 @@ def to_representation(self, value):
 
 
 class MediaSerializer(ExternalSiteContentUrlMixin, serializers.ModelSerializer):
+    # Camel-case these explicitly so they work as inputs as well
```

Comment:
The CamelCase parser that we use has form and multipart versions as well, but they weren't working for me. I solved it this way and logged FW-4650 to deal with all our camelCase. (I noticed some other write APIs have underscores as well.)